compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/input/pointer/PointerEvent.skiko.kt

```diff
@@ -39,7 +37,6 @@ fun PointerButtons(
     return PointerButtons(res)
 }
 
-@ExperimentalComposeUiApi
```

Comment:
This was [upstreamed ](https://android-review.googlesource.com/c/platform/frameworks/support/+/1956205/3/compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/input/pointer/PointerEvent.desktop.kt#27)to androidx-main, so remove experimental